release-notes/6.0/api-diff/.Net/6.0.0_System.md

```diff
@@ -0,0 +1,336 @@
+# System
```

Comment:
Changes under `System.Runtime` generally LGTM. Other APIs should get checked by Tarek, Stephen Toub, and other relevant owners.